extension/ts/colour.ts

```diff
@@ -0,0 +1,26 @@
+const getElement = (document: Document, id: string): HTMLElement =>
+	document.getElementById(id) ??
+	(Array.from(document.getElementsByTagName("frame")) as HTMLIFrameElement[])
+		.map((frame) => getElement(frame.contentWindow.document, id))
+		.find((masthead) => !!masthead);
+
+const editElement = <E extends HTMLElement>(id: string, callback: (element: E) => void) => {
```

Comment:
ooo fancy

Reply:
It's actually bad lmao

I just did an `as` without a type predicate so it's trusting the user to get it right. There is probably a way to get it working properly (I do love pedantic types) but today is not that day

Reply:
ok documented